xml/src/test/java/org/ehcache/xml/XmlConfigurationTest.java

```diff
@@ -772,7 +772,15 @@ public void testCompleteXmlToString() {
     URL resource = XmlConfigurationTest.class.getResource("/configs/ehcache-complete.xml");
     Configuration config = new XmlConfiguration(resource);
     XmlConfiguration xmlConfig = new XmlConfiguration(config);
-    assertThat(xmlConfig.toString(), isSimilarTo(Input.from(resource)).ignoreComments().ignoreWhitespace().withNodeMatcher(new DefaultNodeMatcher(ElementSelectors.byNameAndText)));
+
```

Comment:
This fix is fine. But an even simpler fix would have been to just change the path value in the test xml file to a single directory name without any path separators.